cli/azd/resources/scaffold/templates/host-containerapp.bicept

```diff
@@ -123,6 +136,7 @@ resource app 'Microsoft.App/containerApps@2023-05-02-preview' = {
           allowedOrigins: union(allowedOrigins, [
             // define additional allowed origins here
           ])
+          allowedMethods: ['GET', 'PUT', 'POST', 'DELETE']
```

Comment:
Was this giving you trouble? I was under the impression  that the default value is `allowedMethods: [*]`, but if not, we can make that explicit. I'd prefer  `allowedMethods: [*]` over what we have currently.